debian/tvheadend.default

```diff
@@ -6,6 +6,10 @@
 OPTIONS="-u hts -g video"
 
 # sysvinit
+# 
+# Editing the following variables has no affect when using systemd
```

Comment:
* effect 😄

Reply:
Yep - I realised this mistake not long after committing & forgot to amend!

Thanks for the heads up!